powerhome/playbook
# [PLAY-555] Fixed Body Content disappearing behind Footer for Dialog

#### Screens

## Before:
![Screenshot 2023-01-25 at 4 01 01 PM](https://user-images.githubusercontent.com/73710701/214689769-5f6402f3-010a-4d4b-a5ba-18a6236acc9e.png)

## After:
![Screenshot 2023-01-25 at 3 54 47 PM](https://user-images.githubusercontent.com/73710701/214689567-8937e07b-b4e3-46b1-9464-8e414ce21eb4.png)

#### Breaking Changes

No breaking changes

#### Runway Ticket URL

[Runway Story](https://nitro.powerhrg.com/runway/backlog_items/PLAY-555)

#### How to test this

Click on full height variant in 'scrollable' doc example. Scroll to the bottom to see text is no longer covered by footer.  Test both Rails and React.

#### Checklist:
